superset-frontend/src/views/CRUD/data/dataset/DatasetLayout/DatasetLayout.test.tsx

```diff
@@ -36,18 +36,12 @@ describe('DatasetLayout', () => {
   const mockSetDataset = jest.fn();
 
   const waitForRender = () =>
-    waitFor(() =>
-      render(<Header setDataset={mockSetDataset} datasetName="" />),
-    );
+    waitFor(() => render(<Header setDataset={mockSetDataset} title="" />));
```

Comment:
Isn't this the same as having no title thus the `title: string;` should be optional? Or what implication would that have? 🤔

Reply:
The title can't have nullish values because of the component that uses it: [PageHeaderWithActions](https://github.com/apache/superset/blob/master/superset-frontend/src/components/PageHeaderWithActions/index.tsx)

<img width="707" alt="Screen Shot 2022-09-23 at 4 45 18 PM" src="https://user-images.githubusercontent.com/55605634/192060971-27fa04cc-4ff5-4b84-85ff-4168df8327ab.png">

Reply:
I think the other comments I left might address this and allow the title="" to be removed